### ISSUE TITLE
Snippet-lookin' things are rendered verbatim in the autocompletion popup

### ISSUE THREAD

**Issue by frou** [tomv564/LSP#547]
_Sublime 3203
LSP from Package Control 0.8.2
python-language-server 0.25.0 
macOS 10.14.4_

![Screenshot 2019-03-31 at 11 14 20](https://user-images.githubusercontent.com/172663/55287877-e6996580-53a6-11e9-84e5-d493e27dc146.png)

Is it intended that these snippet things are being presented like that? It makes the autocompletion quite hard to read.

This package has the setting:

```json
// Resolve completions and apply snippet if received.
"resolve_completion_for_snippets": false,
```

I've tried both true and false and it has no effect on the above problem.

BTW, I'm not entirely clear on what that setting does from reading its comment. Could you please describe the difference the user would experience (in terms of what they see and type) between having that enabled and disabled?

**Comment by rwols** [comment]
This is unintended and is being worked on to get fixed.

**Comment by tomv564** [comment]
I think we could introduce another setting where the user can specify if they want LSP to indicate snippet support (as it appears that pyls's snippets just started showing up without the user opting in)

The other setting is because the javascript-typescript language server needed a resolveCompletion call to the server to get the actual snippet. I'd like to get rid of that feature as I suspect few people use it.

**Comment by rwols** [comment]
fixed by #551
